spring-cloud-gcp-samples/spring-cloud-spanner-r2dbc-samples/src/main/java/com/example/Book.java

```diff
@@ -44,11 +45,19 @@ public class Book implements Persistable {
   @Column("CATEGORIES")
   private List<String> categories;
 
+  @Column("COUNT")
+  private int count;
+
+  @Transient
+  private boolean isNew;
+
```

Comment:
Can you just re-use the existing fields. You can add a method like `updateTitle`.

Reply:
wanted to keep the UI part simple, that would need one more text box. let me know if you want that.

Reply:
That's fine.

Reply:
Thanks,